recipes/kalasiris/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "kalasiris" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5bdcb44b71dd7c3a0cf3deb42d18a7516098cb2a372710433413c0d1ac9449db
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  commands:
+    - test -f $PREFIX/kalasiris/kalasiris.py
+  # and then assume that run_test.py gets run here
+
+about:
+  home: http://github.com/rbeyer/kalasiris
+  license: Apache-2.0
```

It's an optional field but it's nice to have:
```suggestion
  license: Apache-2.0
  licence_family: Apache
```